core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -215,6 +215,12 @@ oppia.controller('CollectionPlayer', [
       return completedExplorationIds.indexOf(explorationId) > -1;
     };
 
+    $scope.isNextExploration = function(explorationId) {
+      var nextExplorationId = (
+        $scope.collectionPlaythrough.getNextExplorationIds()[0]);
```

Comment:
Did you test manually? I would have expected some logic along the lines of: if list is empty, return null, otherwise, return the first element of the list.

Also, just to check, is getNextExplorationIds() guaranteed to be sorted (so that the "first" unfinished exploration comes at the front)?